task02_iryna/src/view/SystemInitiator.java

```diff
@@ -0,0 +1,46 @@
+package view;
+
+import controller.TextReader;
+import dao.Paragraph;
+import dao.Sentense;
+import dao.Text;
+import service.TextService;
+
+import java.io.IOException;
+
+public class SystemInitiator {
```

Comment:
View is for UI purposes. In this case combined class. Program flow combined with sout. Consider to split it.